### DISCUSSION TITLE
No more watermark in SD3?

### DISCUSSION THREAD

**Discussion by smandelli** [huggingface/diffusers#9502] [General]
Hi there, I went through Stable Diffusion 3 and noticed that there's no more the option to add a Watermark in generated images (it was so in Stable Diffusion XL). Am I right or the watermark is hidden somewhere else?

**Comment by asomoza** [comment]
cc: @yiyixuxu

**Reply by smandelli** [reply]
Hi @yiyixuxu , can you help me with my previous question? Thanks a lot 👍

**Reply by asomoza** [reply]
Just to add some information before, the watermark had some problems where it added some dots and imperfections to the final image, you can see this [issue](https://github.com/huggingface/diffusers/issues/7841) and this [PR](https://github.com/huggingface/diffusers/pull/7013).

This was added before I was part of the team, personally I never used and I know I wouldn't ever used if I found that it adds those dots to the image which makes them not usable for anything serious. 

Probably the reason it wasn't added was because of time constraints and because it introduces something that worsens the quality.

You can still add it by yourself, as far as I know, it uses this [library](https://github.com/Mamdasn/imWatermark).

I only copied @yiyixuxu to get the official answer to this question.
